third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker.js

```diff
@@ -320,6 +320,35 @@ UserPromptingChecker.prototype.storeUserSelection_ =
 };
 
 /**
+ * Removed stored permission for App with given id.
+ * @param {string} clientAppId
+ */
+UserPromptingChecker.prototype.removeAppPermission = function(clientAppId) {
```

Comment:
I'd suggest to change this method to something like "revokeUserSelection", as it would be better to be able to undo both kinds of the user selections: both "allow" and "deny".

And note that we should also do something with the corresponding method in the PermissionsChecking.Checker class - because, on one side, it's a more generic class, but, on the other side, it's impossible to revoke the policy-granted permissions.